app/react/Metadata/actions/actions.js

```diff
@@ -140,12 +141,13 @@ export function loadInReduxForm(form, _entity, templates) {
         : attachments;
       const entity = { ...response, attachments: sortedAttachments };
       loadFetchedInReduxForm(form, entity, templates).forEach(action => dispatch(action));
+      dispatch(reloadThesauri());
     });
   };
 }
 
 export function changeTemplate(form, templateId) {
-  return (dispatch, getState) => {
+  return async (dispatch, getState) => {
```

Comment:
Is the `async` here because of the `setTimeout` below? I don't understand this change =/